cfgov/jinja2/v1/about-us/the-bureau/about-deputy-director/index.html

```diff
@@ -8,7 +8,7 @@
 {%- endblock %}
 
 {% block desc -%}
-    About David Silberamn
+    David Silberman currently serves as the acting deputy director as well as the associate director of the Research, Markets, and Regulations Division at the Consumer Financial Protection Bureau.
```

Comment:
I was concerned that it would add weird breaks in the description, but I can just test that.